gnucash/import-export/ofx/gnc-ofx-import.c

```diff
@@ -1148,27 +1146,49 @@ gnc_ofx_match_done (GtkDialog *dialog, gpointer user_data)
         return;
     }
 
-    if (info->run_reconcile && info->statement)
+    if (info->run_reconcile && g_list_length (info->statement))
```

Comment:
when testing if a GList has members, it's more efficient to test it against NULL directly.
```c
     if (info->run_reconcile && (info->statement != NULL))
```